lib/conman.rb

```diff
@@ -0,0 +1,50 @@
+require 'console'
+
+class Conman
+
+  def initialize(ui)
+    @ui           = ui
+    @contact_list = []
+  end
+
+  def run
+    answer = "y"
```

Comment:
This string is duplicated knowledge here.
Should it be the responsibility of `Conman` to interpret what a `y` means that a user entered?